src/ostorlab/cli/docker_requirements_checker.py

```diff
@@ -9,7 +9,7 @@
 
 from ostorlab import exceptions
 
-_SUPPORTED_ARCH_TYPES = ["x86_64", "AMD64"]
+_SUPPORTED_ARCH_TYPES = ["x86_64", "AMD64", "arm64"]
```

Comment:
```suggestion
_SUPPORTED_ARCH_TYPES = ["x86_64", "AMD64", "ARM64"]
```

Reply:
`platform.machine()`  is used to compare against this list.